birdhouse/components/cowbird/config/magpie/docker-compose-extra.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.4"
+
+services:
+  # extend Magpie permissions to grant access to Cowbird API via secured Twitcher proxy
+  magpie:
+    links:
+      # must have link to send webhook requests directly though internal network
+      - cowbird
+    volumes:
+      # NOTE:
+      #   Although the file uses the combined "config.yml" format, it is very important to pass it as independent and
+      #   duplicate references for providers/permissions/webhooks config files. This is because 'MAGPIE_CONFIG_PATH' is
+      #   not used to allow parsing additive per-component config files for each extendable service, using loading of
+      #   all configuration files found in mounted directories.
+      - ./components/cowbird/config/magpie/config.yml:/opt/local/src/magpie/config/permissions/cowbird.yml:ro
+      - ./components/cowbird/config/magpie/config.yml:/opt/local/src/magpie/config/providers/cowbird.yml:ro
+      - ./components/cowbird/config/magpie/config.yml:/opt/local/src/magpie/config/webhooks/cowbird.yml:ro
```

Comment:
Multiple files are missing the EOF newline.
I just annotated here to avoid too many comments polluting the PR.

Reply:
@fmigneault I have a better solution for this at #301

Reply:
(I will also update the new files that have no EOF newline for this PR too)

Reply:
Sounds good. I'll add relevant comments directly on that PR.